src/azure/Model/CodeModelTSa.cs

```diff
@@ -77,7 +77,7 @@ public override string ConstructRuntimeImportForModelIndex()
 
         public override string PackageDependencies()
         {
-            return "\"ms-rest-azure-js\": \"~0.13.130\"";
+            return "\"ms-rest-azure-js\": \"/Users/rigibson/src/ms-rest-azure-js\"";
```

Comment:
Run `npm run preview`?

Reply:
Once the dependent PRs for ms-rest-js (and maybe ms-rest-azure-js) are merged